src/commands/CommandRouter.ts

```diff
@@ -0,0 +1,150 @@
+import { Client, Events, GuildScheduledEvent, Interaction, Message, REST } from "discord.js";
+import { RESTPostAPIApplicationCommandsJSONBody, Routes } from "discord-api-types/v9";
+import { Command, CommandDictionary } from "./Command";
+import ReactionCommand from "./reaction/ReactionCommand";
+import { Environment, UserIDs } from "../utils/constants";
+import { isModReaction } from "../utils/helpers";
+import CreateServerEvent from "./reaction/CreateServerEvent";
+import DiscordEventRouter from "../discord/DiscordEventRouter";
+import ContentCommand from "./messageContent/ContentCommand";
+import DiscordBot from "../discord/DiscordBot";
```

Comment:
We should group imports for readability. An idea would be:

1. Library/builtins first (in this particular case, there are none)
2. Imports without curly braces
3. Imports with curly braces

Personally, I would also put a line break between each of the above groups for readability.

Reply:
This is a good reminder that I need to setup some linting rules

Reply:
Went over all files to group them according to these rules.